### PR TITLE
Add Vim for VSCode

### PR DESCRIPTION
**Pull request by jsjoeio**
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,3 +106,11 @@ Templates:
 
 - [testingjavascript.txt](templates/testingjavascript.txt)
 
+### [Vim for VSCode](https://vimforvscode.com) ⌨️
+
+Learn how to use Vim with VSCode
+
+Templates:
+
+- [vim-for-vscode.txt](templates/vim-for-vscode.txt)
```
Comment (kentcdodds) [inline README.md:115]: Awesome! I'm guessing you're working on creating this template?

Comment (jsjoeio) [reply README.md:115]: Either you commented on this before it was added, or I misunderstood 😛

I added it here -> [templates/vim-for-vscode.txt](https://github.com/kentcdodds/kcd-learning-clubs-ideas/blob/2ca4607c36082333c87ecf590cfeedf860d2c208/templates/vim-for-vscode.txt)

File: templates/vim-for-vscode.txt [hunk 2]
```diff
@@ -0,0 +1,26 @@
+Learning Goal: Learn how to use Vim with VS Code
+
+Learning Curriculum: https://vimforvscode.com/
+
+Special Requirements:
+- VSCode
```
Comment (kentcdodds) [inline templates/vim-for-vscode.txt:6]: They also need a license to the course right?

**Review by kentcdodds — COMMENTED** [review]
Awesome! Sorry for the confusion 😅

Just one thing, then this is good to merge.

File: templates/vim-for-vscode.txt (hunk 2 above)
Comment (jsjoeio) [reply templates/vim-for-vscode.txt:6]: oh good call! yes, I will add that in

Comment (jsjoeio) [inline templates/vim-for-vscode.txt:6]:
```suggestion
- VSCode
- a copy of the [course](https://vimforvscode.com/) (PPP supported and student discounts available) 
```

**Review by kentcdodds — APPROVED** [review]
Fantastic! 👏

**Pull request merged**